operators/project.py

```diff
@@ -354,9 +354,15 @@ def on_done(future):
             if isinstance(generated, list):
                 generated = generated[-1]
             if texture is None:
-                texture = bpy.data.images.new(name=str(generated.seeds[0]), width=generated.images[0].shape[1], height=generated.images[0].shape[0])
-            texture.name = str(generated.seeds[0])
-            material.name = str(generated.seeds[0])
+                gen_seed_string = str(generated.seeds[0])
+                prompt_subject = bpy.data.scenes["Scene"].dream_textures_project_prompt.prompt_structure_token_subject
+                name_with_prompt = str(prompt_subject + " " + gen_seed_string)
+                texture = bpy.data.images.new(name=name_with_prompt, width=generated.images[0].shape[1], height=generated.images[0].shape[0])
+            gen_seed_string = str(generated.seeds[0])
+            prompt_subject = bpy.data.scenes["Scene"].dream_textures_project_prompt.prompt_structure_token_subject
```

Comment:
Can you access the scene from the context instead of by name? I think this will fail if the scene is renamed.

Reply:
Resolved in [ca964e3](https://github.com/carson-katri/dream-textures/pull/499/commits/ca964e314dd003ef8318a7838647778a5d406526)